clang/lib/CIR/CodeGen/CIRGenModule.h

```diff
@@ -78,16 +82,37 @@ class CIRGenModule : public CIRGenTypeCache {
 
   void emitTopLevelDecl(clang::Decl *decl);
 
+  /// Return the address of the given function. If funcType is non-null, then
+  /// this function will use the specified type if it has to create it.
+  // TODO: this is a bit weird as `GetAddr` given we give back a FuncOp?
```

Comment:
Both the function name and the comment seem to be wrong. Both appear to be taken from the clang LLVM IR codegen. Should we fix this now? The LLVM IR codegen equivalent returns an `llvm::Constant*` which effectively an address. I'm not sure the same can be said for cir::FuncOp. On the other hand, renaming this will make it difficult to find for anyone porting code from the classic clang codegen.

Reply:
The name of the function and the comment were copied from the incubator project (which, as you noted, copied them from Clang CodeGen).  I want to fix that in the incubator (it's on my list of potential TODOs) where it is easier to see the function in context and to figure out the correct fix.  Trying to fix it as part of upstreaming is, in my opinion, not the best way to fix this.